### PR TITLE
Add Chinese-Tradition(zh_TW) Traslation file

### PR DESCRIPTION
**Pull request by WayneChang65**
Hi Team,
Thanks for the great project.
I just made a Chinese(Traditional) translation.
If it's OK for merge, I'll be glad to be a contributor for this good project.

Wayne

### CODE REVIEW

File: Forms/MainWindow.cpp [hunk 1]
```diff
@@ -125,7 +125,7 @@ void MainWindow::setupModels()
     languageGroup->setExclusive(true);
     connect(languageGroup, &QActionGroup::triggered, this, &MainWindow::slotLanguageChanged);
     currentLanguage = setting.value("settings/locale", "en").toString();
-    QStringList locales = { "de", "en", "es", "fr", "it", "ja", "ko", "zh" };
+    QStringList locales = { "de", "en", "es", "fr", "it", "ja", "ko", "zh", "tw" };
```
Comment (Admiral-Fish) [inline Forms/MainWindow.cpp]: This is causing a runtime crash. There needs to be another QAction added to the QMenu for language for the added language. See the image below to where I am referencing.


image

**Review by Admiral-Fish — CHANGES_REQUESTED** [review]
Thanks for the PR. Just one change that I see.

Also another question. Are things like pokemon names, natures, characteristics, locations, and abilities different from the existing Chinese translations I have? The files for tw are missing from [here](https://github.com/Admiral-Fish/RaidFinder/tree/master/Resources/Text).

File: Forms/MainWindow.cpp (hunk 1 above)
Comment (WayneChang65) [reply Forms/MainWindow.cpp]: Hi Owner, thanks for pointing out the problem.
Actually, I am not familiar with QT and even have not setup the compile environment yet.
Just my friends really need a tw(CHT) Language version RaidFinder, so I tried to translate it in tw(CHT).
Unfortunately, it's not easy as modify string files only and then cause crash. Sorry. 
Now, I added all of tw(CHT) string files and committed them. 
Could you please help me to add code of QAction/QMenu and finish the feature? Thanks a lot.

Wayne

Comment (Admiral-Fish) [reply Forms/MainWindow.cpp]: Yeah I can take care of the rest.

**Pull request merged**